### PR TITLE
283-link-conversion-is-brocken in converter

### LINKED ISSUE #283: [Converter] Link conversion is brocken.
See https://lifesciences.danaher.com/us/en/blog/industrializing-biology-biopharma-4.html

`Prior to COVID-19, the average cost of research and development needed to successfully bring a new drug to market was estimated to be**  [$1335.9 million in the U.S.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054832/), though estimates vary greatly — ranging from ** [$161 million to $4.54 billion](https://pubmed.ncbi.nlm.nih.gov/34368939/)`

Some links still have `**` before and after

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #283 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://283-converter-link-conversion-is-brocken--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
Thank you, looks good.

File: tools/actions/convert/test/fixtures/news2-semantic.html [hunk 1]
```diff
@@ -26,11 +26,11 @@ <h2>ValitaCell Joins Beckman Coulter Life Sciences</h2>
...
...n (NYSE: DHR) family of global science and technology companies. Our mission is to empower those seeking answers to life’s important scientific and healthcare questions. Since 1935, the Beckman name has been synonymous with technologies that simplify and automate complex biomedical testing. Decades later, our global organization also came to embody the scientific legacy of the Coulter name. Today, Beckman Coulter Life Sciences is a trusted, worldwide resource for tools to help optimize research and manufacturing efficiency. Centrifuges. Particle counters/analyzers. Automated liquid handlers. Flow cytometers. Genomic reagents. All these products—and many more—continue to make a difference in people’s lives by improving the productivity of dedicated scientists, quality control experts and others. Wherever people need answers, from prestigious universities and major pharmaceutical companies, to small biotech startups, food/beverage and electronics manufacturing facilities, you can find Beckman Coulter Life Sciences. For more details, visit<a href="https://www.beckman.com/contact-us/?utm_source=dhls_website&#x26;utm_medium=referral&#x26;utm_content=news">beckman.com/contact-us</a>.</p>
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/news2-semantic.html:29]: Is that correct? There is no space between 'visit' and the link?

File: tools/actions/convert/test/fixtures/news2-semantic.html [hunk 2]
```diff
@@ -26,11 +26,11 @@ <h2>ValitaCell Joins Beckman Coulter Life Sciences</h2>
...
       <h2>About ValitaCell</h2>
       <p>ValitaCell’s mission is to accelerate the pace and reduce the cost of manufacturing innovative medicines. We are focused on enabling our biopharmaceutical customers to bring medicines to the patients that need them most, in the shortest length of time. The majority of drugs that advance to clinical trial fail before they make it to the market. This failure increases costs and delays life-saving treatments for patients. Our team is focused on creating analytical technologies that empower scientists with rich cell data and insights earlier in the <a href="/us/en/library/drug-discovery-development-process.html">drug discovery and development</a> process. We are committed to advancing human healthcare worldwide.</p>
       <p>Beckman Coulter, the stylized logo, and the Beckman Coulter product and service marks mentioned herein are trademarks or registered trademarks of Beckman Coulter, Inc. in the United States and other countries.</p>
-      <p><strong>Contact</strong>:Matt McCutcheon, Global External Communications Manager</p>
+      <p>Contact:Matt McCutcheon, Global External Communications Manager</p>
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/news2-semantic.html:33]: Your check below looks ok to me. But I think this strong should not be filtered out?

File: tools/actions/convert/test/fixtures/news2-semantic.html (hunk 1 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/news2-semantic.html:29]: Looks like that was the original content. @davenichols-DHLS , Could you please confirm the same?

File: tools/actions/convert/test/fixtures/news2-semantic.html (hunk 2 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/news2-semantic.html]: Corrected this. Thanks.

**Review by mhaack — APPROVED** [review]

**Pull request merged**